optuna/storages/_rdb/alembic/versions/v3.0.0.b.py

```diff
@@ -14,7 +14,7 @@
 from sqlalchemy import Float
 from sqlalchemy import ForeignKey
 from sqlalchemy import Integer
-from sqlalchemy.orm import declarative_base
+from sqlalchemy.ext.declarative import declarative_base
```

Comment:
`SQLAlchemy==1.4.0` moved `sqlalchemy.ext.declarative.declarative_base` to `sqlalchemy.orm.declarative_base`, and the old one will be deprecated in `SQLAlchemy==2.0.0`.  We need to use the new in future, but I think the current code is acceptable.

https://github.com/sqlalchemy/sqlalchemy/blob/e769ba4b00859ac8c95610ed149da4d940eac9d0/lib/sqlalchemy/ext/declarative/__init__.py#L21-L26

https://github.com/sqlalchemy/sqlalchemy/blob/479dbc99e7fc5a60f846992c0cca8542047a8933/lib/sqlalchemy/util/deprecations.py#L201-L206